blogs/lokiless_netobserv/index.md

```diff
@@ -0,0 +1,91 @@
+
+By: Mehul Modi, Steven Lee
+
+Recently, Network Observability operator 1.6 released a major enhancement to provide network insights for your OpenShift cluster without Loki. This enhancement was also featured in [What's new in Network Observability 1.6](../whats_new_1.6) blog providing quick overview of the feature. In this blog, lets look at some of the advantages and trade-offs users would have when deploying network observability with Loki disabled.
+
+# Configure Network Observability without Loki
+Loki as datasource is currently enabled by default. To configure Network Observability operator without Loki, when configuring Flowcollector resource simply set `.spec.loki.enable` to `false`
+
+```yaml
+loki:
+  enable: false
+```
+
+## Performance and Resource utilization gains
+
+### Query performance:
+<TODO: note faster query performance data when compared to loki here>
+
+### Resource utilization:
+In our tests conducted on 3 different test beds with varied workloads and network throughput, when Network Observability is configured without Loki, total savings of Memory usage could be in range 12-60% and CPU utilzation could be lower by 20-30%<sup>*</sup>. Not to mention you will not need to provision and plan for additional storage in public clouds for Loki, overall reducing the cost and improving operational efficiency significantly.
+
+In our perf tests, [kube-burner](https://github.com/kube-burner/kube-burner) workloads were used to generate several objects and create heavy network traffic. We used sampling rate of 1 for all below tests. To further describe each test bed:
+
+1. Test bed 1: node-density-heavy workload ran against 25 nodes cluster.
+2. Test bed 2: ingress-perf workload ran against 65 nodes cluster.
+3. Test bed 3: cluster-densit-v2 workload ran against 120 nodes cluster
+
+Below graphs shows total vCPU and memory usage for a recommended Network Observability stack (flowlogs-pipeline, eBPF-agent, Kafka and optionally Loki) for production clusters.
+
+![Compare total vCPUs utilized with and without Loki](<images/Total vCPUs consumed.png>)
+![Compare total RSS utilized with and without Loki](<images/Total Memory (RSS) consumed.png>)
+
+<sup>*</sup> actual resource utilization may depend on various factors such as flowcollector sampling size, number of workloads and nodes in an OCP cluster
+
+This comes with couple of trade-offs though, without storage of network flows it no longer provides Traffic flows table. Also, per-pod level of resource granularity is not available since it causes prometheus metrics to have high cardinality. However, should you need per-pod or per-flow level of granularity for diagnostic and troubleshooting needs, enabling loki should be pretty straightforward and both datasources can be used. When both loki and prometheus datasources are enabled, while querying on Netflow Traffic page, prometheus queried data over will be used wherever possible since it offers faster performance.
+
```

Comment:
Maybe there should be a whole section about the tradoff, that includes the previous paragraph about the level of details + the multi-tenancy aspects.

Also speaking of trade-offs, I don't know if you want to dive in these details, but there's also the fact that the metrics model imposes limitations in how we can combine filters/aggregations. For instance, because the per-namespace bandwidth and the per-node bandwidth are provided by two separate metrics, we cannot query with namespace filters and Node scope, or with node filter and Namespace scope. Maybe that's something to mention, which still gives an advantage to Loki in terms of querying flexibility.

Reply:
I have added a separate section for Trade-offs. I think we can leave out filtering/aggregation limitation details, because it's more of a implementation detail which we eventually plan to improve UX for.